src/components/App/App.component.jsx

```diff
@@ -33,6 +33,7 @@ function App() {
   return (
     <BrowserRouter>
       <AuthProvider>
+        <Header />
```

Comment:
maybe you header can be part of your layout

Reply:
Thank you Andres, you are right is cleaner